website/project/model.py

```diff
@@ -4029,8 +4026,8 @@ def add_status_log(self, user, action):
         log = DraftRegistrationLog(action=action, user=user, draft=self)
         log.save()
 
-    def validate_metadata(self, *args, **kwargs):
+    def validate_metadata(self, metadata, reviewer, required_fields):
```

Comment:
It's not intuitive for the delegated `validate_method` to make `reviewer` and `required_fields` required arguments (when they are optional in `Metaschema.validate_metadata`). Since you're just delegating, you can just pass on *args, and **kwargs

```python
def validate_metadata(self, *args, **kwargs):
    return ...validate_metadata(*args, **kwargs)
```